backends/advanced/src/advanced_omi_backend/routers/modules/system_routes.py

```
@@ -128,3 +128,41 @@
+# Memory Configuration Management Endpoints
+
+@router.get("/admin/memory/config/raw")
+async def get_memory_config_raw(current_user: User = Depends(current_superuser)):
+    """Get current memory configuration YAML as plain text. Admin only."""
+    return await system_controller.get_memory_config_raw()
+
+
+@router.post("/admin/memory/config/raw")
+async def update_memory_config_raw(
+    config_yaml: str,
+    current_user: User = Depends(current_superuser)
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)